Source/WebCore/PAL/pal/crypto/commoncrypto/CryptoDigestCommonCrypto.mm

```diff
@@ -26,116 +26,106 @@
 #include "config.h"
 #include "CryptoDigest.h"
 
+#include "PALSwift.h"
 #include <CommonCrypto/CommonCrypto.h>
 
 namespace PAL {
 
 struct CryptoDigestContext {
+    WTF_MAKE_STRUCT_FAST_ALLOCATED;
     CryptoDigest::Algorithm algorithm;
-    void* ccContext;
+    std::variant<
```

Comment:
using contextPointer = std::variant<...>;
That way you don't have to write out the variant types multiple times.